media/js/base/site.js

```diff
@@ -152,6 +153,11 @@
             if (version && parseFloat(version) >= 6.1) {
                 h.className += ' win7up';
             }
+
+            // Check for Windows XP users to alter download button behavior
+            if (/Windows NT 5\.1/i.test(navigator.userAgent)) {
```

Comment:
The [logic from the bouncer efforts](https://github.com/mozilla-services/go-bouncer/pull/16/files#diff-31a4a7824225629df071a1cba8b785d5R26) also looks for "XP". Not sure if this is a real thing, but might be worth checking for as well.